README.md

```diff
@@ -18,17 +17,14 @@ The tool is currently in beta and is available on GitHub here: https://github.co
 - In case any issue is identified, pstress creates a trial directory having all relevant information like stack traces, error logs and thread-wise SQL log files for you to analyze and reproduce the problem.

```

Comment:
In case any issue is identified, it is reported in the pstress run logs. By default, pstress saves each trial directory which has the data directory, server error logs, pstress-thread logs. It also saves the mysqld binaries and the configuration file used for running the test.